rcl/test/rcl/test_init.cpp

```diff
@@ -298,8 +298,30 @@ TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_init_options_acce
   const rcl_allocator_t * options_allocator = rcl_init_options_get_allocator(&init_options);
   EXPECT_TRUE(rcutils_allocator_is_valid(options_allocator));
 
+  size_t domain_id;
+  ret = rcl_init_options_get_domain_id(NULL, &domain_id);
+  ASSERT_EQ(RCL_RET_INVALID_ARGUMENT, ret) << rcl_get_error_string().str;
+  ret = rcl_init_options_get_domain_id(&init_options, NULL);
+  ASSERT_EQ(RCL_RET_INVALID_ARGUMENT, ret) << rcl_get_error_string().str;
+  ret = rcl_init_options_get_domain_id(NULL, NULL);
+  ASSERT_EQ(RCL_RET_INVALID_ARGUMENT, ret) << rcl_get_error_string().str;
+  ret = rcl_init_options_set_domain_id(NULL, domain_id);
+  ASSERT_EQ(RCL_RET_INVALID_ARGUMENT, ret) << rcl_get_error_string().str;
+
+  ret = rcl_init_options_get_domain_id(&init_options, &domain_id);
+  ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
```

Comment:
```suggestion
  EXPECT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
```

all the added `ASSERT_EQ` could also be replaced with `EXPECT_EQ`.

Reply:
> all the added ASSERT_EQ could also be replaced with EXPECT_EQ.

since we do not actually need assertion, replace ASSERT_EQ  into EXPECT_EQ except error expected cases.

Reply:
I keep the code as following,

- API check with ASSERT_EQ.
- Value check (after API) with EXPECT_EQ.